src/main/webapp/charities.html

```diff
@@ -15,24 +16,32 @@
         <ul id="navigation">
             <li><a href="/"><img src="images/home-logo.png" height="25px"></a></li>
             <li><a href="/public-feed.html">Public Feed</a></li>
-            <li><a href="/community.html">Community</a></li>
+            <li><a href="/community.html">Charities Community</a></li>
         </ul>
     </nav>
-    <p>This page allows you to search for charities across the United States and also has a map showing you charities across the country. Hover over a pin to find the name of the charity!</p>
-    <div id="search-container">
+    <div id="logo">
+      <img src="images/logo.png" width="100%">
+    </div>
+	
+    <div id="all-content">
+      <Label id="page-title" class="h1-label">Charities Map and Search</Label>
```

Comment:
the font looks smaller than on other pages, maybe because there's more words. Maybe we can make it just charities ? The little blurb below it tells people that there's a map and search feature

Reply:
Yes, of course! I couldn't figure out why text is smaller! But I'm going to try that